process/block/metablock.go

```diff
@@ -982,6 +982,10 @@ func (mp *metaProcessor) CommitBlock(
 
 	chainHandler.SetCurrentBlockHeaderHash(headerHash)
 
+	if lastMetaBlock != nil && lastMetaBlock.IsStartOfEpochBlock() {
```

Comment:
check.ifNil as lastMetaBock comes from chainHandler and it is of type data.HeaderHandler

Reply:
done